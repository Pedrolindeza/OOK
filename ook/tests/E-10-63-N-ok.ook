public int ook() {
  int x = 0;
  while (x < 30) {
    if (x % 2 == 0) x!!
    x = x + 1;
  }
}

