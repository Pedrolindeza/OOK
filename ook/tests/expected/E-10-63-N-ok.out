0246810121416182022242628
